db/schema.rb

```diff
@@ -67,7 +67,7 @@
     t.string "reagent_kit_barcode", limit: 30, comment: "The barcode for the reagent kit or cartridge"
     t.string "id_flowcell_lims", limit: 20, null: false, comment: "LIMs-specific flowcell id, batch_id for Sequencescape"
     t.integer "position", limit: 2, null: false, comment: "Flowcell lane number", unsigned: true
-    t.string "entity_type", limit: 30, null: false, comment: "Lane type: library, pool, library_control, library_indexed, library_indexed_spike"
+    t.string "entity_type", limit: 30, null: false, comment: "Lane type: library, library_control, library_indexed, library_indexed_spike."
```

Comment:
Need to remove this full stop if you want to remove it from the migration's `up` method.